src/main/ngapp/accio/src/app/img-gallery/img-gallery.component.html

```diff
@@ -23,10 +23,74 @@
   <button type="submit" mat-button  id="upload-button">Upload</button>
 </form>
 
-<div id="gallery" *ngIf="displayImages">
+<br>
```

Comment:
The forms was being difficult with overlapping, I suspect "display: relative". Didn't want to change css scheme that UI team had set with consistency of display attribute, so `<br>` can just be a placeholder if UI team want's to change this overlap correction with css.